MathLib/ODE/OdeSolverTypes.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATHLIB_ODE_ODESOLVERTYPES_H
+#define MATHLIB_ODE_ODESOLVERTYPES_H
+
+#include <Eigen/Core>
+
+namespace MathLib
+{
+template <int M, int N>
+using MappedMatrix = Eigen::Map<Eigen::Matrix<double, M, N, Eigen::ColMajor>>;
```

Comment:
- it's visible also when someone tries to define MappedMatrix at other place in MathLib. It's not easy for others to know that general type is already reserved because it's given as a part of specific functionality.
- i asked about column major because all shape matrices are/must be row major. if we use both ordering types, we should rename them to indicate the type. Sparse matrices are ordering type independent.